docs/blog/authors.yml

```diff
@@ -0,0 +1,17 @@
+endi:
+  name: Endilie Yacop Sucipto
```

Comment:
Who are these people? 😂

Reply:
Authors are a remain of the template, I'll delete them